Nodejs/Product/Nodejs/Project/MigrateToJspsInfoBar.cs

```diff
@@ -94,13 +40,13 @@ public static void Show(IVsShell vsShell, IVsInfoBarUIFactory infoBarUiFactory,
             var infoBarModel = new InfoBarModel(MigrateToJspsResources.MigrateToJspsPrompt, actionItems);
 
             uint eventCookie = 2;
-            Action<string> OnClose = (filePath) =>
+            Action<string> OnClose = (userFile) =>
             {
-                CurrentInfoBarElement.Unadvise(eventCookie);
-                WritePropertyToUserFile(filePath);
+                userPreference.dismissedMigration = true;
+                solutionPersistence.SavePackageUserOpts(userPreference, UserClosedMigrationPrompt);
             };
             CurrentInfoBarElement = infoBarUiFactory.CreateInfoBar(infoBarModel);
-            CurrentInfoBarElement.Advise(new InfoBarUIEvents(OnClose, userFile), out eventCookie);
+            CurrentInfoBarElement.Advise(new InfoBarUIEvents(OnClose, ""), out eventCookie);
```

Comment:
Why is `""` needed? Remove from the `InfoBarUIEvents` ctor and `Action` above.

Reply:
Updated.